src/proto/grpc/testing/stats.proto

```diff
@@ -41,6 +41,12 @@ message ServerStats {
   // change in server time (in seconds) used by the server process and all
```

Comment:
We should scrap sum of squares

On Tue, Mar 29, 2016, 5:26 PM Louis Ryan <notifications@github.com> wrote:

> In src/proto/grpc/testing/stats.proto
> <https://github.com/grpc/grpc/pull/5826#discussion_r57822667>:
>
> > @@ -41,6 +41,12 @@ message ServerStats {
> >    // change in server time (in seconds) used by the server process and all
>
> How is sum-of-squares useful? If we want to calc std deviation we could do
> it from the histogram using a median value for each bucket and the result
> would be pretty accurate.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/5826/files/0b2630be23aeb27286855b65e6133e55949abc7c#r57822667>
>